verilog/tools/kythe/indexing_facts_tree_extractor_test.cc

```diff
@@ -65,14 +66,19 @@ TEST(EqualOperatorTest, FactsTreeExtractor) {
   EXPECT_NE(result_pair4.right, nullptr) << *result_pair4.right;
 }
 
-TEST(ExtractOneFileTest, FactsTreeExtractor) {
+TEST(EmptyCSTTest, FactsTreeExtractor) {
```

Comment:
I would transpose the test name to `TEST(FunctionFeatureBeingTested, TestCaseInputDescription)`